consensus/src/state_computer.rs

```diff
@@ -213,6 +228,16 @@ impl StateComputer for ExecutionProxy {
         // held by BlockExecutor to prevent memory leak.
         self.executor.finish();
 
+        self.async_commit_notifier
```

Comment:
This is interesting. Why do we need this here?

Reply:
Indeed interesting. We added it after debugging an issue with expired transactions. The thing is that we use the latest commit round we are aware of in order to set expiration for the proofs we try to create. If we are behind we set expiration too low and others reject. Here we try to update quorum store with a new commit as soon as we start synching to it.

Reply:
Let's add a comment about this?

Reply:
Done.